modules/i18n/src/translations/en-US/portals/console.ts

```diff
@@ -7570,7 +7570,7 @@ export const console: ConsoleNS = {
                             message: "Something went wrong"
                         },
                         success: {
-                            description: "Successfully deleted the user's association to your organization.",
```

Comment:
Don't we need to alter other languages like French and SInhala?

Reply:
so far we don't have other language support in the invite flow

Reply:
ACK